Description: Remove bundler
 We'll collect dependencies in debian/control
Author: Praveen Arimbrathodiyil <praveen@debian.org>
Last-updated: 2013-05-23

--- a/spec/faked_project/Rakefile
+++ b/spec/faked_project/Rakefile
@@ -1,4 +1,3 @@
-require "bundler"
 
 require "rake/testtask"
 Rake::TestTask.new(:test) do |test|
--- a/spec/faked_project/features/support/env.rb
+++ b/spec/faked_project/features/support/env.rb
@@ -1,4 +1,3 @@
-require "bundler/setup"
 
 # We're injecting simplecov_config via aruba in cucumber here
 # depending on what the test case is...
--- a/spec/faked_project/spec/spec_helper.rb
+++ b/spec/faked_project/spec/spec_helper.rb
@@ -1,4 +1,3 @@
-require "bundler/setup"
 
 # We're injecting simplecov_config via aruba in cucumber here
 # depending on what the test case is...
--- a/spec/faked_project/test/test_helper.rb
+++ b/spec/faked_project/test/test_helper.rb
@@ -1,4 +1,3 @@
-require "bundler/setup"
 
 # We're injecting simplecov_config via aruba in cucumber here
 # depending on what the test case is...
